client/trino-cli/src/test/java/io/trino/cli/TestClientOptions.java

```diff
@@ -367,6 +367,14 @@ public void testDuplicateExtraCredentialKey()
                 .hasMessage("Multiple entries with same key: test.token.foo=bar and test.token.foo=foo");
     }
 
+    @Test
+    public void testMultipleFiles()
+    {
+        Console console = createConsole("--file", "file1.sql", "--file", "file2.sql", "--file", "file3.sql");
+        ClientOptions options = console.clientOptions;
+        assertThat(options.files).isEqualTo(ImmutableList.of("file1.sql", "file2.sql", "file3.sql"));
```

Comment:
Can we add another 2 test cases? 
* file1.sql, file3.sql, file2.sql - to ensure the specified order - not alphabetical
* file1.sql, file1.sql - repeated files